skops/hub_utils/_hf_hub.py

```diff
@@ -213,6 +213,83 @@ def recursively_default_dict() -> MutableMapping:
     dump_json(Path(dst) / "config.json", config)
 
 
+def _create_readme(
```

Comment:
I don't think we need this function.

We can probably simply add such two lines to the `init`:

```py
model_card = card.Card(model, metadata=card.metadata_from_config(Path(local_repo)))
model_card.save(Path(local_repo) / "README.md")
```